README.txt

```diff
@@ -51,7 +54,7 @@ make changes in Sipp, *PLEASE* follow a few coding rules:
     (better portability). (e.g. prefer "printf" to "cout <<" ).
 
   - Check your changes can be compiled on:
- 
+
```

Comment:
If we hadn't changed the files before, we certainly have now! Although I suppose it depends on the legal ramifications of whitespace, something I never thought I'd need to consider.